lib/run_loop/app.rb

```diff
@@ -67,6 +67,22 @@ def executable_name
       identifier
     end
 
+    # Inspects the app's file for the server version
+    def calabash_server_version
+      unless path.nil? || path == ""
+        path_to_bin = File.join(app_path, executable_name)
+        version_str = xcrun.exec(["strings", path_to_bin])[:out][/CALABASH VERSION: \d+\.\d+\.\d+/, 0]
```

Comment:
I would prefer this was split into several lines.

1. Try to keep lines under 80 characters
2. I think there needs to be another check here.  If hash[:out] is nil you will throw an error.  `NoMethodError: undefined method `[]' for nil:NilClass`